### PR TITLE
feat: Set log level via command line flag

### PR DESCRIPTION
**Pull request by DRK3**
Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: cmd/did-rest/startcmd/start_test.go [hunk 1]
```diff
@@ -13,6 +13,8 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/stretchr/testify/require"
+
```
Comment (rolsonquadras) [inline cmd/did-rest/startcmd/start_test.go:16]: extra line

File: cmd/vc-rest/startcmd/start_test.go [hunk 2]
```diff
@@ -13,6 +13,8 @@ import (
 	"os"
 	"testing"
 
+	"github.com/trustbloc/edge-core/pkg/log"
+
```
Comment (rolsonquadras) [inline cmd/vc-rest/startcmd/start_test.go:17]: extra line

File: cmd/did-rest/startcmd/start_test.go (hunk 1 above)
Comment (DRK3) [reply cmd/did-rest/startcmd/start_test.go]: Fixed - good catch.

File: cmd/vc-rest/startcmd/start_test.go (hunk 2 above)
Comment (DRK3) [reply cmd/vc-rest/startcmd/start_test.go]: Fixed - good catch.

**Pull request merged**